runtime/doc/eval.txt

```diff
@@ -1811,7 +1811,36 @@ v:event		Dictionary containing information about the current
 		this dictionary.
 		The dictionary is emptied when the |autocommand|
 		finishes, please refer to |dict-identity| for how to get an
-		independent copy of it.
+		independent copy of it. Example using |deepcopy|: >
+			au TextYankPost * let g:foo = deepcopy(v:event)
+<		Keys vary by event; see the documentation for the specific
+		event, e.g. |CompleteChanged| or |TextYankPost|.
+			KEY		DESCRIPTION ~
+			operator	Current |operator|.  Also set for Ex
+					commands (unlike |v:operator|). For
+					example if |TextYankPost| is triggered
+					by the |:yank| Ex command then
+					`v:event.operator` is "y".
+			regcontents	Text stored in the register as a
+					|readfile()|-style list of lines.
+			regname		Requested register (e.g "x" for "xyy)
+					or the empty string for an unnamed
+					operation.
+			regtype		Type of register as returned by
+					|getregtype()|.
+			completed_item    Current selected complete item on
```

Comment:
Alignment does not look right here (spurious spaces)

Reply:
woops! fixed in latest commit